### ISSUE TITLE
ClickUp embed only works with public links

### ISSUE THREAD

**Issue by alkanna** [outline/outline#5921]
ClickUp embed only seems to be working for public sharings conforming to a specific format.

**To Reproduce**
1. Create a ClickUp embed
2. Paste any link that does not conform to the below regex

**Expected behavior**
Links formatted as app.clickup.com should be compatible

The regex is currently like this :

```
const URL_REGEX = new RegExp(
  "^https?://share.clickup.com/[a-z]/[a-z]/(.*)/(.*)$"
);
```

It does not seem to conform to current private links from ClickUp. Here is an example for a list : https://app.clickup.com/2607968/v/li/204793361

Here is another example for a task :
https://app.clickup.com/t/2607968/SUP-168

**Comment by alkanna** [comment]
@tommoor I've taken a look at this and sadly ClickUp seems to restrict `frame-ancestors 'self'` to a tightly controlled whitelist, and it does not look like this can be changed.

I am pretty convinced that it will not be possible to embed authenticated ClickUp elements, however it might be possible to create custom widgets like notion does (they pull the data from ClickUp API and render a custom element). But that would be a whole new feature !

Long story short, I would recommend closing this as not doable.
